src/lib/lib.h

```diff
@@ -135,8 +135,8 @@ typedef struct _xdebug_func {
 } xdebug_func;
 
 typedef struct xdebug_profile {
-	double        time;
-	double        mark;
+	uint64_t      nanotime;
+	uint64_t      nanotime_mark;
 	long          memory;
```

Comment:
`long` type is not enought for memory stored in bytes, reported under ticket id 1828